### PR TITLE
Add bash script to look up deployment info and commit hash

### PR DESCRIPTION
**Pull request by ChrisPenner**
run `./deployment-info.sh <namespace> <service>` to get info about the code running in your selected k8s config

This can help you look up the source code; config; etc. used to deploy what you currently have running.

Part of the quest to solve https://github.com/wearezeta/backend-issues/issues/742

```
./deployment-info.sh demo brig
image                       release            commit                                    link
quay.io/wire/brig:2.50.314  wire-server-0.1.5  811ced83fe6826de46246622379dbfe56c17e7ec  https://github.com/wireapp/wire-server/releases/tag/image-2.50.314
```

### CODE REVIEW

**Comment by ChrisPenner** [comment]
I've extended it to also have a link to the commit in wire-server-deploy corresponding to the commit which published the chart version which is deployed.

Note that I just changed the tag format; so this script will only work for deployments going forwards from now.

File: README.md [hunk 1]
```diff
@@ -225,3 +226,31 @@ For further configuration options (some have specific requirements about your en
...
+If you wish to get some information regarding the code currently running on your cluster you can run the following:
+
+```
+./bin/deployment-info.sh <namespace> <deployment-name (e.g. brig)>
+```
+
+Example run:
+
+```
+./deployment-info.sh demo brig
```
Comment (ChrisPenner) [inline README.md]: Reminder to chris to update this output with the new fields as soon as a build with the new tag format completes

**Comment by ChrisPenner** [comment]
Pending https://github.com/zinfra/cailleach/pull/210 this will work properly 😄

File: bin/deployment-info.sh [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+# select only docker image tag; not repo
+image_tag="image/$(echo "$image" | cut -f2 -d:)"
+
+wire_server_commit=$(
+    # get all tags from repo
+    git ls-remote --tags "$wire_server_repo" |
+    grep "$image_tag" |
+    cut -f1 |
+    tr -d ' \t\n'
+)
```
Comment (jschaul) [inline bin/deployment-info.sh]: Something here is not quite right, gives empty output for wire_server_commit:

```
./bin/deployment-info.sh staging brig
docker_image:               quay.io/wire/brig:2.50.319
chart_version:              wire-server-0.24.9
wire_server_commit:         
wire_server_link:           https://github.com/wireapp/wire-server/releases/tag/image/image/2.50.319
wire_server_deploy_commit:  01e0f261ca8163e63860f8b2af6d4ae329a32c14
wire_server_deploy_link:    https://github.com/wireapp/wire-server-deploy/releases/tag/chart/wire-server-0.24.9
```

**Pull request merged**